src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.spec.ts

```diff
@@ -65,22 +65,22 @@ describe('Detailed view and edit a selected work item - ', () => {
     fakeUserList = [
       {
         attributes: {
-          fullName: 'Harry Potter',
-          imageURL: 'http://nerdist.com/wp-content/uploads/2016/02/20160210_nerdistnews_harrypottercursedchild_1x1.jpg'
+          fullName: 'WIDCT Example User 0',
```

Comment:
Check line number 590 of this file.  I can't comment there so commenting here. I think this should be **WIDCT Example User 2** instead of **Draco Malfoy**